conjure.schema.json

```diff
@@ -390,7 +390,6 @@
         }
       },
       "required": [
-        "name",
```

Comment:
Seems like circle was green even prior to removing `name` from this required list.  Should we add a test that all of the valid test conjure yml files in https://github.com/palantir/conjure/tree/master/conjure-core/src/test/resources successfully validate against this conjure schema?  Fine to do as a FLUP, IMO